app/weather/weather.service.ts

```diff
@@ -18,6 +18,11 @@ export class WeatherService {
 
     public getCities(url: string) {
         return this.http.get(url)
+            .flatMap((response: Response) => response.json().list);
```

Comment:
with ```flatMap``` you are turning response.json().list into a stream, where each item will be emitted separately

is it the desired behavior?